nikola/plugins/task/sitemap/__init__.py

```diff
@@ -69,6 +70,13 @@
  </sitemap>
 """
 
+alternates_format = """<xhtml:link
```

Comment:
It's copy-pasted from the example in the issue. I don't mind one way or the other :-)